src/components/WwwFrame/Header/KvAtbModalContainer.vue

```diff
@@ -91,6 +92,7 @@ const categoryNames = {
 
 const $kvTrackEvent = inject('$kvTrackEvent');
 const apollo = inject('apollo');
+const cookieStore = inject('cookieStore');
```

Comment:
Is this component used in any stories? I don't think so, but if it is, we'll need to ensure cookie store is available.